### PR TITLE
Updated extension/livolo.js for zigbee-herdsman

### PR DESCRIPTION
**Pull request by ptvoinfo**
Updated extension/livolo.js for zigbee-herdsman

### CODE REVIEW

**Comment by ptvoinfo** [comment]
It seems we should add a test for this extension, but I don't know how to do it :(.

File: lib/extension/livolo.js [hunk 1]
```diff
@@ -60,123 +60,86 @@ class Livolo {
         this.clearTimer();
     }
 
-    _handleCommandRespSimple(err, rsp) {
-        this.ext.configured[this.ieeeAddr].waitresp = false;
-
-        if (err) {
-            if (this.ctype === 'toggle') {
-                logger.debug(`LIVOLO ${this.ieeeAddr}. Toggle command error:`, err.message);
+    async _sendToggle(zdev, ieeeAddr, ep, retry) {
```
Comment (Koenkk) [inline lib/extension/livolo.js]: Why is this here? Doesn't this send a toggle when the users didn't request it?

Comment (ptvoinfo) [reply lib/extension/livolo.js]: This is the special command for LIVOLO, It stops working after a while without this dummy command. The extension sends it only once for each device.

Comment (Koenkk) [reply lib/extension/livolo.js]: I'm thinking if we can move this code to zigbee-shepherd-converters. In this way @kirovilya can also make use of it. 

If we add an extra method to the device definitions in `devices.js` e.g. `onEvent` with events like `started`, `deviceInterview`, etc would this be enough?

Comment (ptvoinfo) [reply lib/extension/livolo.js]: Also, the extension periodically polls each device with the 'read' command. Without this part the converter is useless.

Comment (Koenkk) [reply lib/extension/livolo.js]: in the onEvent code you can call a setInterval so that shouldn't be a problem?

Comment (ptvoinfo) [reply lib/extension/livolo.js]: Do you mean to create a separate timer for each device? Or can I create a global object there for all LIVOLOs?

Comment (Koenkk) [reply lib/extension/livolo.js]: A separate one for each device.

Comment (ptvoinfo) [reply lib/extension/livolo.js]: Then the main problem here is concurrent polling for multiple devices. These unsynced timers may easily overflow commands queue and buffer.

Comment (Koenkk) [reply lib/extension/livolo.js]: I think with zigbee-herdsman that is not a problem because the queuing is properly handled in lower levels.

Comment (ptvoinfo) [reply lib/extension/livolo.js]: Ok. I'll try to move this extensions to converters. Please, let me know when you'll new events.
